roles/core/time/molecule/default/converge.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Converge
+  hosts: all
+
+  tasks:
+    - name: "Include time for servers"
+      include_role:
+        name: "time"
+      vars:
+        - external_time:
```

Comment:
I'm wondering if we should add these values to defaults/main.yml as well?

Reply:
We can, here it would just overwrite the defaults to test another case of needed.